### DISCUSSION TITLE
Will springboot app access newly added plugin without doing restart or build my application ?

### DISCUSSION THREAD

**Discussion by MadhavanAnanthan** [pf4j/pf4j#529] [Q&A]
I want to add or remove a plugin when my application is live, Is it possible my application will access the plugin without doing a maven build or server restart?

**Comment by decebals** [comment]
You can add or remove a plugin without restart the server. I understand the part with restart application (it's what I do in such situations - add or remove plugin) but I don't understand the part with maven build, why would you think of such a thing.
I don't know the behavior in the context of springboot.
In the end, my recommendation is to play a little with a POC app and to share with us any problems that may arise.
Also, take a look on https://github.com/hank-cp/sbp.
